### DISCUSSION TITLE
Error with artFile

### DISCUSSION THREAD

**Discussion by kseniatsocheva** [hemantapkh/PyBirthdayWish#21] [Q&A]
Hi,
When I tried to run the file. I get this error message:

Traceback (most recent call last):
  File "D:\RD30\PyBirthdayWish.py", line 13, in <module>
    art = __import__(f'arts.{artFile}', globals(), locals(), ['*'])
NameError: name 'artFile' is not defined
Could  you please help?

**Comment by hemantapkh** [comment]
The `artFile` is defined in the `config.py` file. Are you sure you did not mess up with `config.py`?
